napari/_qt/qt_main_window.py

```diff
@@ -1205,8 +1235,8 @@ def show(self):
 
     def activate(self):
         """Make the viewer the currently active window."""
-        self._qt_window.raise_()  # for macOS
         self._qt_window.activateWindow()  # for Windows
+        self._qt_window.raise_()  # for macOS
```

Comment:
Was there a reason this order switched?

Reply:
sorry, no, was just testing and forgot to switch back